### PR TITLE
[new release] owl, owl-top and owl-base (1.1)

### PR DESCRIPTION
**Pull request by jzstark**
CHANGES:

* remove several components in owl: `aeos`, `plplot`, `zoo`
* remove the Sparse module in Ndarray and thus Owl's reliance on `eigen`

### CODE REVIEW

File: packages/owl-base/owl-base.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+authors: [ "Liang Wang" ]
+license: "MIT"
+homepage: "https://github.com/owlbarn/owl"
+dev-repo: "git+https://github.com/owlbarn/owl.git"
+bug-reports: "https://github.com/owlbarn/owl/issues"
+doc: "https://owlbarn.github.io/"
+synopsis: "OCaml Scientific and Engineering Computing - Base"
+description: "Owl is an OCaml scientific library."
+build: [
+  [ "dune" "subst" ] {pinned}
```
Comment (kit-ty-kate) [inline packages/owl-base/owl-base.1.1/opam:12]:
```suggestion
  [ "dune" "subst" ] {dev}
```

File: packages/owl-top/owl-top.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+license: "MIT"
+homepage: "https://github.com/owlbarn/owl"
+dev-repo: "git+https://github.com/owlbarn/owl.git"
+bug-reports: "https://github.com/owlbarn/owl/issues"
+doc: "https://owlbarn.github.io/"
+synopsis: "OCaml Scientific and Engineering Computing - Top"
+description: "Owl is an OCaml scientific library."
+
+build: [
+  [ "dune" "subst" ] {pinned}
```
Comment (kit-ty-kate) [inline packages/owl-top/owl-top.1.1/opam:13]:
```suggestion
  [ "dune" "subst" ] {dev}
```

File: packages/owl/owl.1.1/opam [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+description: """
+Owl: OCaml Scientific and Engineering Computing
+
+Owl is an OCaml numerical library.
+It supports N-dimensional arrays, both dense and sparse matrix operations, linear algebra, regressions, fast Fourier transforms, and many advanced mathematical and statistical functions (such as Markov chain Monte Carlo methods).
+Recently, Owl has implemented algorithmic differentiation which essentially makes developing machine learning and neural network algorithms trivial.
+"""
+
+build: [
+  [ "dune" "subst" ] {pinned}
```
Comment (kit-ty-kate) [inline packages/owl/owl.1.1/opam:19]:
```suggestion
  [ "dune" "subst" ] {dev}
```

File: packages/owl/owl.1.1/opam [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  "base-bigarray"
+  "conf-openblas" {>= "0.2.1"}
+  "ctypes" {>= "0.16.0"}
+  "dune" {>= "2.0.0"}
+  "dune-configurator"
+  "owl-base" {= version}
+  "stdio" {build}
+  "npy"
+]
+url {
```
Comment (mseri) [inline packages/owl/owl.1.1/opam:37]:
```suggestion
available: arch = "x86_32" | arch = "x86_34"
url {
```
The following hardcoded flags are not present for other architectures:
```
# gcc: error: unrecognized command-line option '-march=native'; did you mean '-mcpu=native'?
# gcc: error: unrecognized command-line option '-mfpmath=sse'
# gcc: error: unrecognized command-line option '-msse2'
```

**Pull request merged**